TodoList/src/app/screens/Footer/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
```

Comment:
`Footer` is not a screen, it's a Component.
Ask me if you need more instructions on this.